src/checkout-buttons/checkout-button-strategy-action-creator.spec.ts

```diff
@@ -55,29 +45,19 @@ describe('CheckoutButtonStrategyActionCreator', () => {
     });
 
     it('loads required payment method', async () => {
-        const store = createCheckoutStore(getCheckoutStoreState());
-        await Observable.from(strategyActionCreator.initialize(options)(store))
+        await strategyActionCreator.initialize(options)
             .toArray()
             .toPromise();
 
         expect(paymentMethodActionCreator.loadPaymentMethod)
-            .not.toBeCalled();
-    });
-
-    it('does not load payment method if already loaded required payment method', async () => {
```

Comment:
It seems like this is still a valid test to keep. Or at least cover the case of not reloading the payment method if it is already loaded/loading.